packages/nvidia_nat_a365/tests/tooling/test_registration.py

```
@@ -0,0 +1,150 @@
+        config = A365MCPToolingConfig(
+            agentic_app_id="test-agent",
+            auth_token="test-token",
+        )
+
+        mock_builder = Mock()
+
+        # Patch the import to raise ImportError when MCP module is imported
+        # The import happens inside the function: from nat.plugins.mcp.client.client_config import ...
+        original_import = __import__
+
+        def mock_import(name, globals=None, locals=None, fromlist=(), level=0):
```

Ruff (0.15.12)

[error] Function argument `globals` is shadowing a Python builtin

(A002)